gemviz/bluesky_runs_catalog_search.py

```diff
@@ -55,21 +52,9 @@ def getStartTime(uid):
         t_low = min(start_times)
         t_high = max(start_times)
         t_high = utils.ts2iso(utils.iso2ts(t_high) + DAY)
+        
+        self.date_time_widget.setTimeBoundaries(t_low,t_high)
```

Comment:
perhaps `setLimits` instead (shorter, not specific to time, similar to term we use with positioners)